master/usage/decommissioning-a-node.md

```diff
@@ -2,14 +2,21 @@
 title: Decommissioning a Node
 ---
 
-### Why you might be interested in this guide
+### About decommissioning nodes
```

Comment:
Do we want to just nix this header altogether? The title of the page is directly above it and is basically the same thing.

Reply:
I know what you mean, but I like to have a heading for every section. It's more orderly, in my opinion...more organized that way.